src/app/item-popup/DesktopItemActions.tsx

```diff
@@ -349,29 +407,19 @@ function lockButtonTitle(item: DimItem, type: 'lock' | 'track') {
     : t('MovePopup.TrackUntrack.Untracked');
 }
 
-function canShowVault(buttonStore: DimStore, itemOwnerStore: DimStore, item: DimItem): boolean {
-  const store = itemOwnerStore;
-
-  // If my store is the vault, don't show a vault button.
-  // Can't vault a vaulted item.
-  if (!store || store.isVault) {
-    return false;
-  }
-
-  // If my buttonStore is not the vault, then show a vault button.
-  if (!buttonStore.isVault) {
-    return false;
-  }
-
-  // Can't move this item away from the current itemStore.
-  if (item.notransfer) {
-    return false;
-  }
-
-  if (item.location.inPostmaster && !item.canPullFromPostmaster) {
+function canTransferToVault(itemOwnerStore: DimStore, item: DimItem): boolean {
+  if (
+    // item isn't in a store????
```

Comment:
I think this catches stuff like vendor items

Reply:
i would hope action buttons aren't there at that point, but definitely wanted to leave it intact as a condition. it makes sense